orion-server/src/main/java/com/pinterest/orion/core/actions/aws/ReplaceEC2InstanceAction.java

```diff
@@ -152,6 +152,13 @@ public void runAction() {
           );
           getEngine().alert(AlertLevel.MEDIUM, msg);
           getEngine().alert(AlertLevel.HIGH, msg);
+          OrionServer.metricsCounterInc(
+                  "ec2ReplacementGetState",
+                  "failure",
```

Comment:
What is the difference between `failure` and `error`? We used `error` above.

Reply:
If the pagerduty message uses "error", I use error. If it uses "fail", I use failure. I will make them all use error.